src/absil/ilwrite.fsi

```diff
@@ -15,17 +15,22 @@ type ILStrongNameSigner =
     static member OpenKeyPairFile: string -> ILStrongNameSigner
     static member OpenKeyContainer: string -> ILStrongNameSigner
 
+type EmitStreamProvider = Lazy<Choice<EmitTo, Diagnostic>>
```

Comment:
Yes, I think I prefer the second.  I'd imagine the final exposed FCS API will just accept a Stream anyway, won't it, and the ultimate caller of FCS will own closing it.